README.md

```diff
@@ -1,54 +1,35 @@
 Privacy Badger [![Build Status](https://travis-ci.org/EFForg/privacybadger.svg?branch=master)](https://travis-ci.org/EFForg/privacybadger)
 ===================
-Privacy Badger blocks spying ads and invisible trackers. It's there to ensure that companies can't track your browsing without your consent.
+Privacy Badger is a browser extension that automatically learns to block invisible trackers. Instead of keeping lists of what to block, Privacy Badger learns by watching which domains appear to track you as you browse the Web.
```

Comment:
s/appear to track you/appear to be tracking you/

Reply:
Sounds good, anything else?